code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -4,15 +4,14 @@
 #define FAILED_TO_ADD 1
 
 /obj/item/borg/upgrade
-	name = "A borg upgrade module."
-	desc = "Protected by FRM."
+	name = "robot upgrade"
+	desc = "Protected by Firmware Rights Management."
 	icon = 'icons/obj/module.dmi'
 	var/locked = FALSE
 	var/list/required_module = list()
```

Comment:
I'm mildly annoyed that it's a list and it's not plural.

Reply:
/shrug